grails-app/controllers/org/pih/warehouse/api/StockMovementApiController.groovy

```diff
@@ -104,7 +112,13 @@ class StockMovementApiController {
      */
     def updateShipment = { //StockMovement stockMovement ->
         StockMovement stockMovement = stockMovementService.getStockMovement(params.id)
-        bindStockMovement(stockMovement, request.JSON)
+        try {
```

Comment:
I don't mind adding these here, but it seems like this would already happen within the ErrorsController since the exception thrown by bindStockMovement would propagate to the ErrorsController and be rendered properly there. Was there something that made you decide to handle the errors within each of the actions?

Reply:
You're right. I just automaticaly coverd it with try catch, because I forgot about ErrorController.